tools/apidiff/merger.cs

```diff
@@ -76,22 +76,25 @@ public static void Process (string platform, string path, string os)
 		headers.WriteLine ($"# {title}");
 		headers.WriteLine ();
 
-		File.WriteAllText (filename, headers.ToString ());
+		var filePath = Path.Combine (destinationPath, filename);
+		File.WriteAllText (filePath, headers.ToString ());
 
 		var alldiffs = content.ToString ();
 		if (alldiffs.Length == 0)
 			alldiffs = "No changes were found between both versions."; // should not happen for releases (versions change)
-		File.AppendAllText (filename, alldiffs);
-		Console.WriteLine ($"@MonkeyWrench: AddFile: {Path.GetFullPath (filename)}");
+		File.AppendAllText (filePath, alldiffs);
+		Console.WriteLine ($"@MonkeyWrench: AddFile: {Path.GetFullPath (filePath)}");
 
-		if (File.Exists ("api-diff.html"))
-			File.AppendAllText ("api-diff.html", $"\n<h2><a href=\"{filename}\">{platform} API diff (markdown)</a></h2>");
+		if (File.Exists ("api-diff.html")){
+			// TODO changing filename to filePath may cause trouble here...
+			File.AppendAllText ("api-diff.html", $"\n<h2><a href=\"{filePath}\">{platform} API diff (markdown)</a></h2>");
+		}
 	}
 
 	public static int Main (string [] args)
 	{
 		try {
-			Process (args [0], args [1], args [2]);
+			Process (args [0], args [1], args [2], args [3]);
 			return 0;
 		} catch (Exception e) {
 			Console.WriteLine (e);
```

Comment:
This changes are not needed, right? Since it is just for the links in the index, this locally will be a problem

Reply:
Hmm on line 16 of this file, I added a parameter to the Process method.
Are you saying that other places call this method and won't have this fourth argument?

Reply:
I'm asking, why are we changing the links since we are not going to be using the index.html page?

Reply:
@mandel-macaque this last argument brings the path that we want to write the markdown file to since before, it was just using the current working directory. Since we now how the markdowns for the current PR diffs, we need to be more specific with the paths.